Explorer/Assets/DCL/Rendering/Highlight/HighlightInputRenderPass.cs

```diff
@@ -0,0 +1,192 @@
+﻿using DCL.Diagnostics;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace DCL.Rendering.Highlight
+{
+    public partial class HighlightRendererFeature : ScriptableRendererFeature
+    {
+        private static readonly int highlightColour = Shader.PropertyToID("_HighlightColour");
+        private static readonly int outlineWidth = Shader.PropertyToID("_Outline_Width");
+        private static readonly int highlightObjectOffset = Shader.PropertyToID("_HighlightObjectOffset");
+
+        public class HighlightInputRenderPass : ScriptableRenderPass
```

Comment:
Minor thing, but the fields and params in the class does not follow naming standards. We dont use snake_case on fields (only consts), nor `_` prefix on params